test/coverage/CMakeLists.txt

```diff
@@ -76,32 +76,24 @@ add_custom_command(
     COMMENT "Open ${PROJECT_BINARY_DIR}/html/index.html in your browser to view the coverage report."
 )
 
-# overload seqan3_test to fit
-macro (seqan3_test target_cpp)
-    # $target_cpp = "pod_tuple_test.cpp"
-    #   * will register the global TARGET name "pod_tuple_test" and
-    #   * will register the test case name "core_pod_tuple" if
-    #     pod_tuple_test.cpp is in test/core/
-    #
-    # NOTE(marehr): ".+/test/" REGEX is greedy, that means
-    # /test/test/test/hello_test.cpp will result in an empty `test_path`
-    string (REGEX REPLACE "_test.cpp$" "" target_name ${target_cpp})
-    string (REGEX REPLACE ".+/test/" "" test_path ${CMAKE_CURRENT_LIST_DIR})
-    set (target "${target_name}_coverage_test")
-
-    add_executable (${target} ${target_cpp})
+macro (seqan3_test unit_test_cpp)
+    file (RELATIVE_PATH unit_test "${CMAKE_SOURCE_DIR}/.." "${CMAKE_CURRENT_LIST_DIR}/${unit_test_cpp}")
```

Comment:
Out of the three macros for performance, coverage and unit, only this one has the `/..` after souce dir ("${CMAKE_SOURCE_DIR}`/..`") why ?

Reply:
thank you! should have been `"${CMAKE_SOURCE_DIR}/../unit"`, because this is the base dir of the test cases.